ci/scripts/execute_tests.sh

```diff
@@ -69,13 +69,17 @@ fi
 
 case $ARTIFACT_SLUG in
   windows*)
+    echo "Making environment adjustments for windows."
     JAVA_BUILD_PATH=C:/java${JAVA_BUILD_VERSION}
     JAVA_TEST_PATH=C:/java${JAVA_TEST_VERSION}
+    GRADLE_SKIP_TASK_OPTIONS="${GRADLE_SKIP_TASK_OPTIONS} -x docker"
```

Comment:
does this imply that docker stuff works under OpenJDK but not under Bellsoft JDK?